packages/fluentui/react-bindings/src/hooks/useAutoControlled.ts

```diff
@@ -24,6 +24,10 @@ export const useAutoControlled = <Value>(
   // Used to avoid dependencies in "setValue"
   const valueRef = React.useRef(value);
 
+  React.useEffect(() => {
+    valueRef.current = value;
+  }, [value]);
```

Comment:
let's test this, `@testing-library/react-hooks` is now available from the entire repo but you can also use the existing tests too

WDYT @layershifter ?